website/docs/gitops-templates/templates.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Using templates
+sidebar_position: 3
+hide_title: true
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Using templates <TierLabel tiers="enterprise" />
+
+## Orgnizing Templates
+
+Declare the type of a template by using the `weave.works/template-type` label. The value of the label is the name of the template type. The template type is used to group templates in the UI.
+
+Recommended template types:
+- `application` - for application templates
+- `cluster` - for cluster templates
+- `terraform` - for Terraform templates
+
+## Enabling/Disabling Template Components
+
+Enable or disable rendering of certain component sections in a template with the use of annotations. This can be done by using the `templates.weave.works/COMPONENT-enabled` annotation with a boolean value.
+
+Supported components:
+- `profiles`
+- `kustomizations`
+- `credentials`
+
+Example:
+```
+annotations:
+  templates.weave.works/profiles-enabled: true
+  templates.weave.works/kustomizations-enabled: true
+  templates.weave.works/credentials-enabled: true
+```
+
+## Rendering Templates
+
+Declare the render type indicating the templating language to be used to render the template by using the `renderType` spec.
+
+Supported templating languages:
+- `envsubst` - uses [envsubst](https://github.com/a8m/envsubst) for rendering 
+- `templating` - uses text/templating for rendering supporting the following [functions](http://masterminds.github.io/sprig/):
+  - String Functions: *trim*, *wrap*, *randAlpha*, *plural*
+    - String List Functions: *splitList*, *sortAlpha*
+  - Integer Math Functions: *add*, *max*, *mul*
+    - Integer Slice Functions: *until*, untilStep
+  - Float Math Functions: *addf*, *maxf*, *mulf*
+  - Date Functions: *now*, *date*
+  - Defaults Functions: *default*, *empty*, *coalesce*, *fromJson*, *toJson*, *toPrettyJson*, *toRawJson*, ternary
+  - Encoding Functions: *b64enc*, *b64dec*
+  - Lists and List Functions: *list*, *first*, *uniq*
+  - Dictionaries and Dict Functions: *get*, *set*, *dict*, *hasKey*, *pluck*, *dig*, *deepCopy*
+  - Type Conversion Functions: *atoi*, *int64*, *toString*
+  - Flow Control Functions: *fail*
+  - Advanced Functions
+    - UUID Functions: *uuidv4*
+    - Version Comparison Functions: *semver*, semverCompare
+    - Reflection: *typeOf*, *kindIs*, *typeIsLike*
+
+
+## How to: Add a CAPI Template
+
+CAPI Templates objects need to be wrapped with the `CAPITemplate` custom resource and then loaded into the management cluster.
+
+```yaml
+apiVersion: capi.weave.works/v1alpha1
+kind: CAPITemplate
```

Comment:
Update any references go GitOpsTemplate instead so it is consistent across our docs.